docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
     container_name: database
     image: mysql:8.0
     platform: linux/x86_64     # for Windows or Mac Intel Chips
```

Comment:
Selam @agitrubard belki burayı ihtiyaç halinde kullanılması için yorum satırı içine alabiliriz. Benim geliştirme ortamımda(Win11 + WSL2 enable) bu satır yorum satırında da çalıştı. Emeğine sağlık ✨

Reply:
Selam @emreylmaz,
Burası sadece WSL2 enable iken çalışıyor diye anlayıp burayı yorum satırına aldım 🙌🏼